automated/linux/kselftest/kselftest.sh

```diff
@@ -107,6 +108,20 @@ while getopts "i:n:c:T:t:s:u:p:L:S:b:g:e:h" opt; do
         e)
             export ENVIRONMENT="${OPTARG}"
             ;;
+        E)
+            if [ -n "$OPTARG" ]; then
+                 IFS='=' read -r env_name env_val << EOF
+$OPTARG
+EOF
```

Comment:
I think this would be enough.

IFS='=' read -r env_name env_val <<< "${OPTARG}"

Reply:
yeah i had that initially, but shellcheck warns on that

Reply:
OK understood, thank you.